cli/pcluster/cfnconfig.py

```diff
@@ -382,6 +385,9 @@ def __check_account_capacity(self):  # noqa: C901
                 MaxCount=max_size,
                 ImageId=test_ami_id,
                 SubnetId=subnet_id,
+                CpuOptions={"CoreCount": 1, "ThreadsPerCore": 1}
```

Comment:
Get this from the `self.parameters.get("ComputeCoreCount")`

Reply:
Done